pgorod/SuiteCRM
# HTML Emails fix

Fixes several problems with the Email templates:

## Description
- translated languages were showing tags in Email subjects, like `<p>`
- translation process had to be manual because extra HTML tags needed to be added in templates, but couldn't, because they would render incorrectly in emails
- HTML was being removed from Emails, unnecessarily reducing the ability to make better looking templates
- links in emails weren't showing as links, but as visible URL's

## Motivation and Context


## How To Test This
- Try the actions that generate emails (assignments, invitations, reminders), and see the effect on emails in several email clients (I used the online Gmail, Thunderbird, and a very old webmail client circa 2002). Check the email source to see what's being sent, not just what's being rendered.
- upload the templates into Crowdin translation platform and check if strings appear as translatable entities, and translations don't break the HMTL templates (horus68 already confirmed this for us).

## Types of changes


### Final checklist
